src/net/sourceforge/kolmafia/request/PingRequest.java

```diff
@@ -117,14 +115,6 @@ protected boolean shouldFollowRedirect() {
     return false;
   }
 
-  public long getStartTime() {
```

Comment:
Again, why did you delete these "getters"?

Reply:
It's unused as far as I can tell.